lib/galaxy/tool_util/client/staging.py

```diff
@@ -60,24 +74,20 @@ def _handle_job(self, job_response):
 
     def stage(
         self,
-        tool_or_workflow,
-        history_id,
-        job=None,
-        job_path=None,
```

Comment:
job_path is used by Planemo - I would appreciate preserving this.

Reply:
Restored, thanks for catching it!